test/orchard/java/classpath_test.clj

```diff
@@ -22,22 +22,29 @@
 (deftest classpath-test
   (testing "Classpath"
     (testing "URLs are absolute file paths"
-      (is (every? #(.isAbsolute (File. (.getPath %)))
```

Comment:
Changed all instances of `is every?` to `is` inside a `doseq`. 

This way:

* the precise thing that failed is reported
* we avoid huge failure reports, especially now that we added the `"unzipped-jdk-source"` resource path

Reply:
Agreed!

Reply:
Btw, I'd even suggest adding this as a guideline to the `Clojure Style Guide` under "Testing".